config/config-observability.yaml

```diff
@@ -19,79 +19,84 @@ metadata:
-    # Parse json log before sending to Elastic Search
-    <filter **>
-      @type parser
-      key_name log
-      <parse>
-        @type multi_format
-        <pattern>
-          format json
-          time_key fluentd-time # fluentd-time is reserved for structured logs
-          time_format %Y-%m-%dT%H:%M:%S.%NZ
-        </pattern>
-        <pattern>
-          format none
-          message_key log
-        </pattern>
-      </parse>
-    </filter>
-    # Send to Elastic Search
-    <match **>
-      @id elasticsearch
-      @type elasticsearch
-      @log_level info
-      include_tag_key true
-      # Elasticsearch service is in monitoring namespace.
-      host elasticsearch-logging.knative-monitoring
-      port 9200
-      logstash_format true
-      <buffer>
-        @type file
-        path /var/log/fluentd-buffers/kubernetes.system.buffer
-        flush_mode interval
-        retry_type exponential_backoff
-        flush_thread_count 2
-        flush_interval 5s
-        retry_forever
-        retry_max_interval 30
-        chunk_limit_size 2M
-        queue_limit_length 8
-        overflow_action block
-      </buffer>
-    </match>
+  # logging.fluentd-sidecar-output-config: |
+  #   # Parse json log before sending to Elastic Search
+  #   <filter **>
+  #     @type parser
+  #     key_name log
+  #     <parse>
+  #       @type multi_format
+  #       <pattern>
+  #         format json
+  #         time_key fluentd-time # fluentd-time is reserved for structured logs
+  #         time_format %Y-%m-%dT%H:%M:%S.%NZ
+  #       </pattern>
+  #       <pattern>
+  #         format none
+  #         message_key log
+  #       </pattern>
+  #     </parse>
+  #   </filter>
+  #   # Send to Elastic Search
+  #   <match **>
+  #     @id elasticsearch
+  #     @type elasticsearch
+  #     @log_level info
+  #     include_tag_key true
+  #     # Elasticsearch service is in monitoring namespace.
+  #     host elasticsearch-logging.knative-monitoring
+  #     port 9200
+  #     logstash_format true
+  #     <buffer>
+  #       @type file
+  #       path /var/log/fluentd-buffers/kubernetes.system.buffer
+  #       flush_mode interval
+  #       retry_type exponential_backoff
+  #       flush_thread_count 2
+  #       flush_interval 5s
+  #       retry_forever
+  #       retry_max_interval 30
+  #       chunk_limit_size 2M
+  #       queue_limit_length 8
+  #       overflow_action block
+  #     </buffer>
+  #   </match>
 
   # The revision log url template, where ${REVISION_UID} will be replaced by the actual
   # revision uid. For the url to work you'll need to set up monitoring and have access to
   # the kibana dashboard using `kubectl proxy`.
-  logging.revision-url-template: |
-    http://localhost:8001/api/v1/namespaces/knative-monitoring/services/kibana-logging/proxy/app/kibana#/discover?_a=(query:(match:(kubernetes.labels.knative-dev%2FrevisionUID:(query:'${REVISION_UID}',type:phrase))))
+  # logging.revision-url-template: |
```

Comment:
This is also something vendor specific and each vendor will probably want to put their own values here rather than letting customer override it.

Reply:
If vendors don't want this specified, then they can set it and `kubectl apply` will start to overwrite it.